modules/persistent-storage-local-install.adoc

```diff
@@ -49,3 +49,85 @@ $ oc annotate project local-storage openshift.io/node-selector=''
+
+.. Navigate to *Operators* -> *OperatorHub*.
+
+.. Type *Local Storage* into the filter box to locate the Local Storage Operator.
+
+.. Click *Install*.
+
+.. On the *Create Operator Subscription* page, select *A specific namespace on the cluster*. Select *local-storage* from the drop-down menu.
+
+.. Adjust the values for the *Update Channel* and *Approval Strategy* to the desired values.
+
+.. Click *Subscribe*.
+
+. Once finished, the Local Storage Operator will be listed in the *Installed Operators* section of the web console.
+
+.From the CLI
+. Install the Local Storage Operator from the CLI.
+
+.. Create an object YAML file to define a Namespace, OperatorGroup, and Subscription for the Local Storage Operator,
+such as `local-storage.yaml`:
++
+.Example local-storage
+[source,yaml]
+----
+  apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: local-storage
+  ---
+  apiVersion: operators.coreos.com/v1alpha2
+  kind: OperatorGroup
+  metadata:
+    name: local-operator-group
+    namespace: local-storage
+  spec:
+    targetNamespaces:
+      - local-storage
+  ---
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: local-storage-operator
+    namespace: local-storage
+  spec:
+    channel: "4.2"
+    installPlanApproval: Automatic
+    name: local-storage-operator <1>
+    source: redhat-operators <2>
+    sourceNamespace: openshift-marketplace <3>
+----
+<1> Name of the Operator to subscribe to.
+<2> Name of the CatalogSource that provides the Operator.
+<3> Namespace of the CatalogSource. Use `openshift-marketplace` for the default
+OperatorHub CatalogSources.
+
+. Create the Local Storage Operator object by entering the following command:
++
+----
+$ oc apply -f local-storage.yaml
+----
++
+At this point, the Operator Lifecycle Manager (OLM) is now aware of the Local Storage Operator. A ClusterServiceVersion (CSV) for the Operator should appear in the target namespace, and APIs provided by the Operator should be available for creation.
++
+. Verify local storage installation by checking that all Pods and the Local Storage Operator have been created:
+
+.. Check that all the required Pods have been created:
++
+----
+$ oc -n local-storage get pods
+NAME                                      READY   STATUS    RESTARTS   AGE
+local-storage-operator-746bf599c9-vlt5t   1/1     Running   0          19m
+----
+
+.. Check the ClusterServiceVersion (CSV) YAML manifest to see that the Local Storage Operator is available in the `local-storage` project:
++
+----
+$ oc get csvs -n local-storage
+NAME                                         DISPLAY         VERSION               REPLACES   PHASE
+local-storage-operator.4.2.26-202003230335   Local Storage   4.2.26-202003230335              Succeeded
+----
+
+After all Pods are ready and you have verified that the Local Storage Operator is available, you can begin using local storage to store data from your cluster.
```

Comment:
After the local storage operator is deployed, we still need to create `localvolume` object to install the local storage provisioner to help provide PVs.

Reply:
> After the local storage operator is deployed, we still need to create `localvolume` object to install the local storage provisioner to help provide PVs.

This procedure (creating local volume resource) is described in the next section immediately following Operator install: 
https://docs.openshift.com/container-platform/4.4/storage/persistent_storage/persistent-storage-local.html#local-volume-cr_persistent-storage-local

Reply:
@bobfuru got it.  `you can begin using local storage to store data from your cluster` misleads me, how about change to something like `After all the check passed, local storage operator is installed successfully.`

Reply:
@qinpingli Sure - that makes sense. I changed to your suggestion.